llvm/labs/CMakeLists.txt

```diff
@@ -7,4 +7,4 @@ ENDFOREACH()
 
 set(LLVM_TEST_DEPENDS ${LLVM_TEST_DEPENDS} PARENT_SCOPE)
 
-message("LLVM_TEST_DEPENDS ${LLVM_TEST_DEPENDS}")
+message("LLVM_TEST_DEPENDS ${LLVM_TEST_DEPENDS}")
```

Comment:
> Please revert this change as not necessary.

I'm sorry I don't know how. But it doesn't change anything so why not just leave it like that?

Reply:
```suggestion
message("LLVM_TEST_DEPENDS ${LLVM_TEST_DEPENDS}")

```

Reply:
> > Please revert this change as not necessary.
> 
> I'm sorry I don't know how. But it doesn't change anything so why not just leave it like that?

Seems that your changes removed `\n` char in the end of last line. Actually, it is required to add new line symbol to the last line of any file in the LLVM project.
Regarding, accidental changes if they even don't affect anything it may complicate analysis of changes in a project if there are mess of non-related changes in a file.

Reply:
> > > Please revert this change as not necessary.
> > 
> > 
> > I'm sorry I don't know how. But it doesn't change anything so why not just leave it like that?
> 
> Seems that your changes removed `\n` char in the end of last line. Actually, it is required to add new line symbol to the last line of any file in the LLVM project. Regarding, accidental changes if they even don't affect anything it may complicate analysis of changes in a project if there are mess of non-related changes in a file.

Does that mean that I should add extra `\n` on 3 files that has been created by me (and start github actions again)? Or I just should be careful about it on the next labs? Because I didn't put extra `\n` in the first lab's files.

Reply:
Yes, it would be nice of you to add a new line in the end of all your files in the code.